syntax = "proto3";
package garden;

message HeartBeatRequest {
    uint64 request_timestamp_ms = 1;
}

message HeartBeatReply {
    uint64 timestamp_ms = 1;
    float cpu_temperature_degC = 6;
}

service Garden {
    rpc HeartBeat (HeartBeatRequest) returns (HeartBeatReply);
}   